stable/rabbitmq/templates/NOTES.txt

```diff
@@ -1,54 +1,61 @@
 
-  RabbitMQ can be accessed within the cluster on port {{ .Values.rabbitmq.nodePort }} at {{ template "rabbitmq.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
+RabbitMQ can be accessed within the cluster on port {{ .Values.rabbitmq.nodePort }} at {{ template "rabbitmq.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
 
-  To access for outside the cluster execute the following commands:
+To access for outside the cluster, perform the following steps:
 
 {{- if contains "NodePort" .Values.serviceType }}
 
+Obtain the NodePort IP and ports:
+
     export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
     export NODE_PORT_AMQP=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[1].nodePort}" services {{ template "rabbitmq.fullname" . }})
     export NODE_PORT_STATS=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[3].nodePort}" services {{ template "rabbitmq.fullname" . }})
 
-  To Access the RabbitMQ AMQP port:
+To Access the RabbitMQ AMQP port:
 
-    echo amqp://$NODE_IP:$NODE_PORT_AMQP/
+    URL : amqp://$NODE_IP:$NODE_PORT_AMQP/
 
-  To Access the RabbitMQ Management interface:
+To Access the RabbitMQ Management interface:
 
-    echo http://$NODE_IP:$NODE_PORT_STATS/
+    URL : http://$NODE_IP:$NODE_PORT_STATS/
 
 {{- else if contains "LoadBalancer" .Values.serviceType }}
 
-  NOTE: It may take a few minutes for the LoadBalancer IP to be available.
-        Watch the status with: 'kubectl get svc --namespace {{ .Release.Namespace }} -w {{ template "rabbitmq.fullname" . }}'
+Obtain the LoadBalancer IP:
+
+NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+      Watch the status with: 'kubectl get svc --namespace {{ .Release.Namespace }} -w {{ template "rabbitmq.fullname" . }}'
 
     export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ template "rabbitmq.fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
 
-  To Access the RabbitMQ AMQP port:
+To Access the RabbitMQ AMQP port:
 
-    echo amqp://$SERVICE_IP:{{ .Values.rabbitmq.nodePort }}/
+    URL : amqp://$SERVICE_IP:{{ .Values.rabbitmq.nodePort }}/
 
-  To Access the RabbitMQ Management interface:
+To Access the RabbitMQ Management interface:
 
-    echo http://$SERVICE_IP:{{ .Values.rabbitmq.managerPort }}/
+    URL : http://$SERVICE_IP:{{ .Values.rabbitmq.managerPort }}/
 
 {{- else if contains "ClusterIP"  .Values.serviceType }}
 
-   kubectl port-forward {{ template "rabbitmq.fullname" . }}-0  --namespace {{ .Release.Namespace }} {{ .Values.rabbitmq.nodePort }}:{{ .Values.rabbitmq.nodePort }} {{ .Values.rabbitmq.managerPort }}:{{ .Values.rabbitmq.managerPort }}
+Create a proxy server between localhost and the Kubernetes API Server:
+
+    kubectl proxy --port 8002 &
```

Comment:
Not sure if we should put the command to run in the background

Reply:
I suggested background in order to be able to copy paste the commands on the same terminal. I'm open to remove it.